pkg/security/podsecuritypolicy/v1beta1/factory.go

```diff
@@ -22,16 +22,16 @@ import (
 	utilfeature "k8s.io/apiserver/pkg/util/feature"
 	"k8s.io/kubernetes/pkg/features"
 
+	corev1 "k8s.io/api/core/v1"
+	policyv1beta1 "k8s.io/api/policy/v1beta1"
 	"k8s.io/apimachinery/pkg/util/errors"
-	api "k8s.io/kubernetes/pkg/apis/core"
-	"k8s.io/kubernetes/pkg/apis/policy"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/apparmor"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/capabilities"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/group"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/seccomp"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/selinux"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/sysctl"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/user"
+	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/v1beta1/apparmor"
```

Comment:
unless we intend to have multiple versions of the podsecuritypolicy utilities (which I don't think we do), let's leave these in the existing location. as PSP graduates to v1, we would wait N releases, then switch the admission plugin to use v1

Reply:
make sense. also, keeping the same location makes the pull much smaller.